src/components/atomic/VFormLabel.vue

```diff
@@ -1,35 +1,41 @@
 <template>
-  <div class="relative mt-4">
-      <label :for=title class="text-base leading-7 text-blueGray-500">
-          {{ labelTitle }}
-      </label>
-      <slot></slot>
-      <v-help-text :help=help ></v-help-text>
-      <div v-if="errors" v-for="error in errors" :key="error.id">
-          <v-error :error=error ></v-error>
-      </div>
-  </div>
+    <div class="relative mt-4">
+        <label :for=title class="text-base leading-7 text-blueGray-500" v-if="title">
+            {{ title }} <span v-if="required">* <span class="sr-only">Required</span></span>
+        </label>
+        <slot></slot>
+        <v-help-text v-if="hasHelp" :help=help></v-help-text>
+        <div v-if="errors" v-for="error in errors" :key="error.id">
+            <v-error :error=error></v-error>
+        </div>
+    </div>
 </template>
 
 <script>
+import VHelpText from './VHelpText';
+import VError from './VError';
+
 export default { // Wrapper Component for Inputs which shows Tooltip and Help text, also will need to handle errors
-  name: "VFormLabel",
-  props: {
-    "title": { type: String, required: true },
-    "required": { type: Boolean, required: false, default: false },
-    "help": { type: String, required: true },
-    "tooltip": { type: String, required: true },
-    "errors": { type: String, required: false, default: '' }
-  },
-  computed: {
-      labelTitle()
-      {
-          return this.required ? this.title : this.title + ' *';
-      }
-  }
+    name: "VFormLabel",
+    components: {
+        VHelpText,
+        VError
+    },
+    props: {
+        title: {type: String, required: false},
+        required: {type: Boolean, required: false, default: false},
+        help: {type: String, required: false},
```

Comment:
Help text should be required ideally, as you don't want to make inputs that don't have anything to help the user.

Reply:
It will definitely nearly always be put on. I'm thinking for things like a checkbox to say 'I've read the T&Cs', or more relevant a checkbox to say 'I've read through the transaction lists for this year'. I don't think that'd benefit from a help text at all, and making it required means we'd have to have one